catroid/src/main/java/org/catrobat/catroid/pocketmusic/mididriver/MidiInstruments.java

```diff
@@ -0,0 +1,158 @@
+	public static final byte LEAD_2_CALLIOPE                = 82;
+	public static final byte LEAD_3_CHIFF                   = 83;
+	public static final byte LEAD_4_CHARANG                 = 84;
+	public static final byte LEAD_5_VOICE                   = 85;
+	public static final byte LEAD_6_FIFTHS                  = 86;
+	public static final byte LEAD_7_BASS_LEAD               = 87;
+	public static final byte PAD_0_NEW_AGE                  = 88;
+	public static final byte PAD_1_WARM                     = 89;
+	public static final byte PAD_2_POLYSYNTH                = 90;
+	public static final byte PAD_3_CHOIR                    = 91;
+	public static final byte PAD_4_BOWED                    = 92;
+	public static final byte PAD_5_METALLIC                 = 93;
+	public static final byte PAD_6_HALO                     = 94;
+	public static final byte PAD_7_SWEEP                    = 95;
+	public static final byte FX_0_RAIN                      = 96;
+	public static final byte FX_1_SOUNDTRACK                = 97;
+	public static final byte FX_2_CRYSTAL                   = 98;
+	public static final byte FX_3_ATMOSPHERE                = 99;
+	public static final byte FX_4_BRIGHTNESS                = 100;
+	public static final byte FX_5_GOBLINS                   = 101;
+	public static final byte FX_6_ECHOES                    = 102;
+	public static final byte FX_7_SCI_FI                    = 103;
+	public static final byte SIT_R                          = 104;
+	public static final byte BANJO                          = 105;
+	public static final byte SHAMISEN                       = 106;
+	public static final byte KOTO                           = 107;
+	public static final byte KALIMBA                        = 108;
+	public static final byte BAG_PIPE                       = 109;
+	public static final byte FIDDLE                         = 110;
+	public static final byte SHANAI                         = 111;
+	public static final byte TINKLE_BELL                    = 112;
+	public static final byte AGOGO                          = 113;
+	public static final byte STEEL_DRUMS                    = 114;
+	public static final byte WOODBLOCK                      = 115;
+	public static final byte TAIKO_DRUM                     = 116;
+	public static final byte MELODIC_TOM                    = 117;
+	public static final byte SYNTH_DRUM                     = 118;
+	public static final byte REVERSE_CYMBAL                 = 119;
+	public static final byte GUITAR_FRET_NOISE              = 120;
+	public static final byte BREATH_NOISE                   = 121;
+	public static final byte SEASHORE                       = 122;
+	public static final byte BIRD_TWEET                     = 123;
+	public static final byte TELEPHONE_RING                 = 124;
+	public static final byte HELICOPTER                     = 125;
+	public static final byte APPLAUSE                       = 126;
+	public static final byte GUNSHOT                        = 127;
```

Comment:
The formatter doesn't like horizontal aligning :scream_cat:. You can use
```
// @formatter:off
...
// @formatter:on
```
to disable the formatter and Checkstyle (https://github.com/Catrobat/Catroid/commit/d599f93b8282119017ca5483a0001f5bc7001626). Please use it sparely!

Enums are really powerful. They can store constants and also have methods. See [here (Planets)](http://docs.oracle.com/javase/tutorial/java/javaOO/enum.html).
If you still prefer numbers, take a look at [enum-annotation](https://developer.android.com/studio/write/annotations.html#enum-annotations). I'm not sure if it works with bytes though.